examples/error_handler_middleware.rs

```diff
@@ -0,0 +1,83 @@
+use tide::{Request, Response, Result, StatusCode};
+#[async_std::main]
+async fn main() -> tide::Result<()> {
+    tide::log::with_level(tide::log::LevelFilter::Trace);
+    let mut app = tide::new();
+
+    app.middleware(ErrorHandler::new(
```

Comment:
Is there a good way to make this take any arbitrary number of these, i.e. a slice of closures or something?

Reply:
I think it's probably clearer to just have a bunch of app.middleware()'s, especially since that allows them to be interleaved with other middlewares